tests/baselines/reference/spreadOfParamsFromGeneratorMakesRequiredParams.errors.txt

```diff
@@ -1,8 +1,6 @@
-error TS2318: Cannot find global type 'IterableIterator'.
 spreadOfParamsFromGeneratorMakesRequiredParams.ts(6,1): error TS2554: Expected 2 arguments, but got 1.
 
 
-!!! error TS2318: Cannot find global type 'IterableIterator'.

```

Comment:
This error only crops up while resolving one of the overloads (the iterator one, ofc), which isn't the last one that we "accept" (though it still has an error), so it gets tossed.